src/pages/Article.jsx

```diff
@@ -0,0 +1,59 @@
+import { useParams } from "react-router-dom"
+import { useEffect, useState } from "react"
+import { getArticlesData } from "../../app"
+import { getSpecficArticleData } from "../../app";
+
+
+
+function Article() {
+
+    const {article_id} = useParams();
+    const [article, setArticle] = useState(null);
```

Comment:
I would recommend using an empty object as the default value of this state instead of null